manim/mobject/opengl_compatibility.py

```diff
@@ -0,0 +1,31 @@
+from abc import ABCMeta
+
+from .. import config
+
+# from .mobject import Mobject # cannot import since this is cyclic
```

Comment:
I don't think it is necessary to keep this comment around.

```suggestion
```

Reply:
I think now that `Mobject` and `VMobject` don't depend on `ConvertToOpenGL`, you could add this back in actually and use `Mobject` and `VMobject` directly for  the conversion instead of their names, which I would prefer

Reply:
There are some additional classes in `vectorized_mobject` for which the metaclass has to be imported, the cyclic import isn't fully resolved yet, unfortunately.

I'd actually like to generously restructure a lot of the modules, maybe we should delay cleaning this up until then?

Reply:
Sounds good to me